submissions/natashafir/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,49 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+function Inhabitants(species, name, gender, legs, hands, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
+    this.saying = saying;
+    this.sayHi = function() {
```

Comment:
The proper way to assign functions as methods in JS is via using an object's prototype. This approach (a) ensures proper inheritance when building hierarchies of classes, (b) decerases memory consuption when every new object is being created.
Check [this](https://medium.com/@chamikakasun/javascript-prototype-and-prototype-chain-explained-fdc2ec17dd04) out.